gems/pending/util/miq-xml.rb

```diff
@@ -5,43 +5,34 @@
 require 'util/xml/xml_diff'
 require 'util/xml/xml_patch'
 
-class MiqXml  
-	#MIQ_XML_VERSION = 1.0
-	#MIQ_XML_VERSION = 1.1	# Added create_time to root in seconds for easier time conversions
-	MIQ_XML_VERSION = 2.0	# Changed sub-xmls, added namespaces
+class MiqXml
+  MIQ_XML_VERSION = 2.0 # Changed sub-xmls, added namespaces
 
-	@@defaultXmlType = :rexml  # REXML is always available so default it here.
+  @@defaultXmlType = :rexml  # REXML is always available so default it here.
 
-  # Now test to see if nokogiri is available
+  # Set to true if nokogiri is set as parser.
   @@nokogiri_loaded = false
-	begin

```

Comment:
@chessbyte Ultimately this is a good first pass at fixing the current MiqXml API with respect to nokogiri, so that we *can* drop Rexml in favor of nokogiri and then eventually drop MiqXml entirely.